linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_groupownership/ansible/shared.yml

```diff
@@ -13,14 +13,14 @@
   ansible.builtin.set_fact:
     local_users: '{{ ansible_facts.getent_passwd|dict2items }}'
 
-- name: Test for existence home directories to avoid creating them, but only fixing ownership
+- name: Test for existence of home directories to avoid creating them, but only fixing ownership
   ansible.builtin.stat:
     path: '{{ item.value[4] }}'
   register: path_exists
   loop: '{{ local_users }}'
   when:
-    - item.value[2]|int >= {{{ gid_min }}}
-    - item.value[2]|int != 65534
+    - item.value[1]|int >= {{{ uid_min }}}
+    - item.value[1]|int != {{{ nobody_uid }}}
```

Comment:
small nit pick: it might be useful for future readers to add a comment about type of data at index 1 (and at index 4 above) in the array

Reply:
Done